### ISSUE TITLE
Improve accordion with summary line semantics and presentation when JS loads but CSS doesn't 

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#2327]
## What
We should consider the semantics and presentation of the accordion with the summary line if the CSS fails to load and JS doesn't. As was pointed out at the [accessibility clinic](https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-906449543), this is what a screen reader user will hear when navigating the component. 

This is also unlikely to be considered a good arrangement of content:
>“If you showed this to a content designer and said ‘is this good content for a button’, I’m pretty sure they would say it’s not.”

This is related to https://github.com/alphagov/govuk-frontend/issues/2336 - changing the design so that the button doesn't contain the summary line is likely to fix this issue. 

If we don't change the design, the visual presentation when JS loads but CSS doesn't could be improved by not transforming the sections to buttons if CSS hasn't loaded - but this is will not help with the semantics when JS does load.

## Why

We should ensure that the component follows good practise and makes sense semantically. Also, in the somewhat edge case where JS loads but CSS doesn't, the buttons end up with a large amount of content that isn't visually ordered.

## Who needs to know about this
Developer, designer

## Done when

- [ ] We've either resolved https://github.com/alphagov/govuk-frontend/issues/2336 which will hopefully help with this card too, or considered (and created a card for) a potential alternative solution

**Comment by hannalaakso** [comment]
## General improvements

We're going to mitigate against related issues raised at the accessibility clinic, see 'Next steps' https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-922840762 for full write up.

## Improve the semantics of the summary in the button by adding punctuation
Another thing we should try to do is to **insert a comma after the summary line in the markup**. This is because we had the slightly unexpected feedback from the accessibility clinic that the punctuation symbol which was originally introduced to improve NVDA announcements (https://github.com/alphagov/govuk-frontend/pull/2257/commits/83c8b570d4f2101ccc39126e1d35e3ed0ca59886 - if the link breaks with rebasing, it's the commit titled "Add pause") also improves the semantics of the button by dividing up its content thematically, as demonstrated here:
![Screenshot 2021-09-20 at 14 47 57](https://user-images.githubusercontent.com/5007934/134014521-c14d0546-8e6f-4c80-b344-77328e5f52b3.png)

## Next steps
I'm going to leave this issue open for now so that we can track the work to:
- add a comma character after the summary line markup with JavaScript, and test that change in screen readers
- check if we can remove the non-breaking space character after the existing comma (as seen in the above screenshot), it seems redundant

**Comment by hannalaakso** [comment]
This now renders as the following shows which is a slight improvement:
![Screenshot 2021-10-05 at 15 36 36](https://user-images.githubusercontent.com/5007934/136044587-628c8e21-e679-496c-861d-fa9ac3943730.png)

**Comment by hannalaakso** [comment]
In addition, the decision to add the commas in was based on a previous investigation done by GOV.UK Publishing Frontend: https://accessibility.blog.gov.uk/2017/12/18/what-working-on-gov-uk-navigation-taught-us-about-accessibility/
